extensions/extensions-helm/aissemble-infrastructure-chart/tests/namespaces_test.yaml

```diff
@@ -0,0 +1,91 @@
+suite: infrastructure
+templates:
+  - namespaces.yaml
+tests:
+  - it: Should contain a Argo CD namespace
+    documentIndex: 0
+    asserts:
+      - containsDocument:
+          kind: Namespace
+          apiVersion: v1
```

Comment:
will the document also have the name info? maybe we can also assert `name: argocd` like in the below test

Reply:
Yes that can be done too!